desc/compute/_bootstrap.py

```diff
@@ -0,0 +1,504 @@
+            )
+        )
+    )
+
+    # Redl eq (13):
+    F32ee = (
+        (0.1 + 0.6 * Zeff_rho)
+        * (X32e - X32e**4)
+        / (Zeff_rho * (0.77 + 0.63 * (1 + (Zeff_rho - 1) ** 1.1)))
+        + 0.7
+        / (1 + 0.2 * Zeff_rho)
+        * (X32e**2 - X32e**4 - 1.2 * (X32e**3 - X32e**4))
+        + 1.3 / (1 + 0.5 * Zeff_rho) * (X32e**4)
+    )
+
+    # Redl eq (16):
+    X32ei = f_t / (
+        1
+        + 0.87 * (1 + 0.39 * f_t) * jnp.sqrt(nu_e) / (1 + 2.95 * (Zeff_rho - 1) ** 2)
+        + 1.53 * (1 - 0.37 * f_t) * nu_e * (2 + 0.375 * (Zeff_rho - 1))
+    )
+
+    # Redl eq (15):
+    F32ei = (
+        -(0.4 + 1.93 * Zeff_rho)
+        / (Zeff_rho * (0.8 + 0.6 * Zeff_rho))
+        * (X32ei - X32ei**4)
+        + 5.5
+        / (1.5 + 2 * Zeff_rho)
+        * (X32ei**2 - X32ei**4 - 0.8 * (X32ei**3 - X32ei**4))
+        - 1.3 / (1 + 0.5 * Zeff_rho) * (X32ei**4)
+    )
+
+    # Redl eq (12):
+    L32 = F32ei + F32ee
+
+    # Redl eq (19):
+    L34 = L31
+
+    # Redl eq (20):
+    alpha0 = (
+        -(0.62 + 0.055 * (Zeff_rho - 1))
+        * (1 - f_t)
+        / (
+            (0.53 + 0.17 * (Zeff_rho - 1))
+            * (1 - (0.31 - 0.065 * (Zeff_rho - 1)) * f_t - 0.25 * f_t * f_t)
+        )
+    )
+    # Redl eq (21):
+    alpha = (
+        (alpha0 + 0.7 * Zeff_rho * jnp.sqrt(f_t * nu_i)) / (1 + 0.18 * jnp.sqrt(nu_i))
+        - 0.002 * nu_i * nu_i * (f_t**6)
+    ) / (1 + 0.004 * nu_i * nu_i * (f_t**6))
+
+    # Factor of elementary_charge is included below to convert temperatures from eV to J
+    dnds_term = (
+        -G
+        * elementary_charge
+        * (ne_rho * Te_rho + ni_rho * Ti_rho)
+        * L31
+        * (d_ne_d_s / ne_rho)
+        / (psi_edge * (iota - helicity_N))
+    )
+    dTeds_term = (
+        -G
+        * elementary_charge
+        * pe_rho
+        * (L31 + L32)
+        * (d_Te_d_s / Te_rho)
+        / (psi_edge * (iota - helicity_N))
+    )
+    dTids_term = (
+        -G
+        * elementary_charge
+        * pi_rho
+        * (L31 + L34 * alpha)
+        * (d_Ti_d_s / Ti_rho)
+        / (psi_edge * (iota - helicity_N))
+    )
+    J_dot_B = dnds_term + dTeds_term + dTids_term
+
+    # Store all results in the J_dot_B_data dictionary:
+    nu_e_star = nu_e
+    nu_i_star = nu_i
+    variables = [
+        "rho",
+        "ne_rho",
+        "ni_rho",
+        "Zeff_rho",
+        "Te_rho",
+        "Ti_rho",
+        "d_ne_d_s",
+        "d_Te_d_s",
+        "d_Ti_d_s",
+        "ln_Lambda_e",
+        "ln_Lambda_ii",
+        "nu_e_star",
+        "nu_i_star",
+        "X31",
+        "X32e",
+        "X32ei",
+        "F32ee",
+        "F32ei",
+        "L31",
+        "L32",
+        "L34",
+        "alpha0",
+        "alpha",
+        "dnds_term",
+        "dTeds_term",
+        "dTids_term",
+    ]
+    J_dot_B_data = geom_data.copy()
+    for v in variables:
+        J_dot_B_data[v] = eval(v)
+    J_dot_B_data["<J*B>"] = J_dot_B
+
+    if plot:
```

Comment:
could this be moved to a standalone function in `desc.plotting`?

Reply:
For simplicity I've removed this plotting section. I haven't used it lately, and probably others wouldn't use it enough to be worth maintaining it.